tests/functional/m/method_hidden_py39.py

```diff
@@ -0,0 +1,142 @@
+# pylint: disable=too-few-public-methods,missing-docstring
+# pylint: disable=unused-private-member
+# pylint: disable=import-error
+"""check method hiding ancestor attribute
+"""
+import functools as ft
+import something_else as functools
+
+
+class Abcd:
+    """dummy"""
+
+    def __init__(self):
+        self.abcd = 1
+
+
+class Cdef(Abcd):
+    """dummy"""
+
+    def abcd(self):  # [method-hidden]
+        """test"""
+        print(self)
```

Comment:
You can actually delete most of this file. It only needs to contain the one test that doesn't run on Python 3.8. All the other tests in the other file will still run on all versions.

(This also has the benefit of clarifying which tests have to do purely with your PR.)